### ISSUE TITLE
Port 5060 is now in Chrome unsafe port list

### ISSUE THREAD

**Issue by moqmar** [DanielBok/private-conda-repo#13]

**Comment by DanielBok** [comment]
You can change the port exposed. For example in the snippet linked, change `5060:5060` to `8080:5060`

https://github.com/DanielBok/private-conda-repo/blob/master/server/config.yaml#L35

**Comment by moqmar** [comment]
Yeah, I got it to work, but the issue is more about it not working anymore with the default configuration.

For everyone who wants a quick fix, you additionally need to re-build the "web" container with `REACT_APP_API_URL` set to the correct URL, you need to add that as an `ARG REACT_APP_API_URL=http://xyz.local:5055` to the Dockerfile.
It's used here:
https://github.com/DanielBok/private-conda-repo/blob/e9c5dd8f333664abeb7fc417368ab7b3b8c6d43e/web/src/infrastructure/api/index.ts#L13-L15
